MSAL/src/MSALPublicClientApplication.m

```diff
@@ -139,6 +139,7 @@ + (void)load
     MSIDNotifications.webAuthDidReceiveResponseFromBrokerNotificationName = MSALWebAuthDidReceiveResponseFromBroker;
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->